docs/topics/functions.md

```diff
@@ -127,37 +121,32 @@ reformat(
 )
 ```
 
-You can skip all the ones with default values:
+You can disregard all of the ones with default values:
 
 ```kotlin
 reformat("This is a long String!")
 ```
 
-You are also able to skip specific arguments with default values, rather than omitting them all. However, after the first
-skipped argument, you must name all subsequent arguments:
+You can also skip specific arguments with default values rather than omitting all of them. After the first skipped argument, however, you must name all subsequent arguments:
```

Comment:
```suggestion
You can also skip specific arguments with default values, rather than omitting them all. However, after the first skipped argument, you must name all subsequent arguments:
```
It is clearer for the reader if **however** is at the start of the sentence, to make it clear that this is an exception.